compiler/qsc_partial_eval/tests/calls.rs

```diff
@@ -1285,7 +1285,7 @@ fn call_to_unresolved_callee_producing_dynamic_value_fails() {
 
     assert_error(
         &error,
-        &expect!["UnexpectedDynamicValue(Span { lo: 298, hi: 305 })"],
+        &expect!["UnexpectedDynamicValue(PackageSpan { package: PackageId(2), span: Span { lo: 298, hi: 305 } })"],
```

Comment:
Could you add a test case there that calls into something from stdlib that will fail with `PackageId(1)` to confirm that behavior?

Reply:
I'll add one :)

Reply:
Added test.